libfreerdp/crypto/certificate.c

```diff
@@ -621,10 +641,14 @@ BOOL certificate_data_print(rdpCertificateStore* certificate_store,
 		return FALSE;
 	}
 
-	size = _snprintf(NULL, 0, "%s %"PRIu16" %s %s %s\n", certificate_data->hostname,
+	rc = _snprintf(NULL, 0, "%s %"PRIu16" %s %s %s\n", certificate_data->hostname,
 	                 certificate_data->port,
 	                 certificate_data->fingerprint, certificate_data->subject,
 	                 certificate_data->issuer);
+	if (rc < 0)
+		return FALSE;
+	size = (size_t)rc;
```

Comment:
This cast looks wrong (size is of type UINT64)

Reply:
good catch, missed that one.